one-ui/src/main/java/com/marklogic/hub/curation/controllers/JobsController.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2012-2019 MarkLogic Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package com.marklogic.hub.curation.controllers;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.marklogic.hub.FlowManager;
+import com.marklogic.hub.impl.FlowManagerImpl;
+import com.marklogic.hub.job.JobDocManager;
+import com.marklogic.hub.oneui.managers.SearchableManager;
+import com.marklogic.hub.oneui.models.HubConfigSession;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.Collections;
+import java.util.List;
+
+@Controller
+@RequestMapping(value="/api/jobs")
+public class JobsController extends SearchableManager {
+    @Autowired
+    private HubConfigSession hubConfig;
+
+
+    @RequestMapping(method = RequestMethod.GET)
+    @ResponseBody
+    public ResponseEntity<?> getJobs(@RequestParam(value = "flowName", required = false) String flowName) {
+        JobDocManager jobDocManager = getJobDocManager();
```

Comment:
Would this be appropriate in JobDocManager, or is this logic too specific to the definition of "getJobs" here?